shared/fs/footer/upload.desktop.js

```diff
@@ -35,28 +35,40 @@ const realCSS = `
   top: ${height}px;
   transition: all .3s ${easing};
 }
+.text-overflow {
+  color: ${globalColors.white};
+  max-width: 60vw;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-align: center;
+}
```

Comment:
Is there a way to do this in the `styles` object below? I think we prefer to not use real CSS whenever possible.

I'm not good a CSS, but for example would `max-width: '100%'` work here?

Reply:
I'd have to change a lot for that to work, `max-width: 100%` won't work since `flexbox` styles override `position` styles

Reply:
Also, the reason I moved this from the object below is because of a flow error, turns outs the only way to make `text-overflow` work is by passing the css directly

Reply:
we should not move things into the 'real' css blocks because of this. The reason this doesn't pass on flow is because its not actually cross platform with RN. This is what the platformStyles helper is for

Reply:
Thanks for pointing that out, how do I address this to make it work for cross-platform?

Reply:
Thanks, let me add that.